crates/bevy_render/src/pipeline/render_pipelines.rs

```diff
@@ -1,12 +1,13 @@
 use super::{PipelineDescriptor, PipelineSpecialization};
 use crate::{
-    draw::{Draw, DrawContext},
+    draw::{Draw, DrawContext, Viewable},
     mesh::{Indices, Mesh},
     prelude::{Msaa, Visible},
     renderer::RenderResourceBindings,
 };
 use bevy_asset::{Assets, Handle};
 use bevy_ecs::{
+    prelude::With,
```

Comment:
```suggestion
    With,
```

Reply:
that does not compile for me.

Reply:
![image](https://user-images.githubusercontent.com/24433899/110235275-73ec7980-7f2f-11eb-899a-ffe9de0288cd.png)
seems to be the only option here.

Reply:
it's `bevy_ecs::query::With` now.